pyiron_base/jobs/job/extension/server/queuestatus.py

```diff
@@ -295,29 +301,36 @@ def update_from_remote(project, recursive=True, ignore_exceptions=False):
         else:
             jobs_now_running_lst = []
         failed_jobs = []
-        for job_id in df_combined.id.values:
-            if job_id not in jobs_now_running_lst:
-                try:
-                    job = project.inspect(job_id)
-                    state.queue_adapter.transfer_file_to_remote(
-                        file=job.project_hdf5.file_name,
-                        transfer_back=True,
-                        delete_remote=False,
-                    )
-                    status_hdf5 = job.project_hdf5["status"]
-                    project.set_job_status(job_specifier=job.job_id, status=status_hdf5)
-                    if status_hdf5 in job_status_finished_lst:
-                        job_object = job.to_object()
-                        job_object.transfer_from_remote()
-                except Exception as e:
-                    if ignore_exceptions:
+        fetch_ids = df_combined.id.values[
+            ~np.isin(df_combined.id.values, df_queue.jobid.values)
```

Comment:
```suggestion
            np.isin(df_combined.id.values, df_queue.jobid.values, invert=True)
```

If I'm reading [the docs](https://numpy.org/doc/stable/reference/generated/numpy.isin.html) right, this is equivalent but apparently faster.